libs/dataset-explorer/src/lib/FeatureBalanceMeasuresChart.tsx

```diff
@@ -0,0 +1,232 @@
+                        selectors: {
+                          ".ms-Button-flexContainer": {
+                            width: "100%"
+                          }
+                        }
+                      },
+                      dropdown: {
+                        width: 150
+                      }
+                    }}
+                    id="featureDropdown"
+                    options={featureOptions}
+                    selectedKey={this.state.selectedFeatureIndex}
+                    onChange={this.setSelectedFeature}
+                  />
+                </Stack.Item>
+              </Stack>
+            </Stack.Item>
+
+            {/* Renders the Measure dropdown and its heading */}
+            <Stack.Item>
+              <Stack tokens={{ childrenGap: "s1" }}>
+                <Stack.Item>
+                  <Text variant="mediumPlus" className={styles.boldText}>
+                    {measuresLocalization.MeasurePicker}
+                  </Text>
+                </Stack.Item>
+                <Stack.Item>
+                  <Dropdown
+                    styles={{
+                      callout: {
+                        selectors: {
+                          ".ms-Button-flexContainer": {
+                            width: "100%"
+                          }
+                        }
+                      },
+                      dropdown: {
+                        width: 200
+                      }
+                    }}
+                    id="measureDropdown"
+                    options={measureOptions}
+                    selectedKey={this.state.selectedMeasureIndex}
+                    onChange={this.setSelectedMeasure}
+                  />
+                </Stack.Item>
+              </Stack>
+            </Stack.Item>
+
+            {/* Renders the description of what user chose from dropdowns */}
+            <Stack.Item>
+              <Stack tokens={{ childrenGap: "s1" }}>
+                <Stack.Item>
+                  {/* Empty placeholder so that description lines up with dropdowns */}
+                  <br />
+                </Stack.Item>
+                <Stack.Item>
+                  <Text>
+                    {/* Because <Text> does not support bolding single words, split it into multiple <Text>s
+                        Format is: "Showing <measure> gaps on all classes of <feature>" */}
+                    <Text variant="mediumPlus">
+                      {measuresLocalization.Description1}
+                    </Text>
+                    <Text variant="mediumPlus" className={styles.boldText}>
+                      {` ${selectedMeasure} `}
+                    </Text>
+                    <Text variant="mediumPlus">
+                      {measuresLocalization.Description2}
+                    </Text>
+                    <Text variant="mediumPlus" className={styles.boldText}>
+                      {` ${selectedFeature}`}
+                    </Text>
```

Comment:
I see the problem you faced here. Are we sure that the word order will translate the same way to all other supported languages? If not, then localization will make this very weird. I can't say I have a good solution. @xuke444 maybe?

Reply:
@xuke444 Do you happen to have an idea for how to tackle this?

Reply:
I kept this method for now. I am also concerned about the word order in other languages. I actually switched the `boldText` `<Text>` objects to `<Label>`, but the idea is the same. Its a bunch of `<Text>`s and `<Label>`s one after another.